BDArmory/Bullets/PooledBullet.cs

```diff
@@ -746,6 +795,16 @@ bool BulletHitAnalysis(RaycastHit hit, float period)
                 ME.massMod += massMod;
                 ME.duration += BDArmorySettings.WEAPON_FX_DURATION;
             }
+            if (EMP)
+            {
+                var emp = hitPart.vessel.rootPart.FindModuleImplementing<ModuleDrainEC>();
+                if (emp == null)
+                {
+                    emp = (ModuleDrainEC)hitPart.vessel.rootPart.AddModule("ModuleDrainEC");
+                } 
+                emp.incomingDamage += (caliber * Mathf.Clamp(bulletMass - tntMass, 0.1f, 101)); //soft EMP caps at 100; can always add a EMP amount value to bulletcfg later, but this should work for now
+                emp.softEMP = true;
```

Comment:
We may want to review bullet EMP damage at somepoint, but this should work for now